Documentation/subcommands/cp.md

```diff
@@ -0,0 +1,58 @@
+# rkt cp
+
+Copy a file or directory from or to a running pod, given the pod UUID.
+
+Files on the host can be referenced using absolute or relative paths. Files inside a pod are referenced using `UUID:ABSOLUTE_PATH`. Files can be copied from the host to a pod or the other way around. Copying files between pods is not supported. The destination path must reference an existing directory. File permissions are preserved on copy. Symlinks are not followed and skipped.
```

Comment:
The 'destination must reference a directory' seems surprising for copying individual files (e.g. `rkt cp $uuid:/bin/bash /tmp/my-container-bash` seems like something people would expect to work).

The symlink behaviour also seems a little odd. There might be a better option than ignoring symlinks? Especially for copying a single file I think there can be a better option